src/ImageInTerminal.jl

```diff
@@ -93,6 +97,8 @@ function __init__()
         @warn "ImageInTerminal is not supported for Windows platform: Julia at least v1.6.0 is required."
         disable_encoding()
     end
+
+    pushdisplay(TerminalGraphicDisplay(stdout))
```

Comment:
This might not be ideal because if there're other GUI image viewers, we'd better use them. Currently, it seems that vscode is smart enough to figure the order out, I'll need to check other GUIs to see if it's also the same case.

```julia
julia> typeof.(Base.Multimedia.displays)
4-element Vector{DataType}:
 TextDisplay
 REPL.REPLDisplay{REPL.LineEditREPL}
 ImageInTerminal.TerminalGraphicDisplay{Base.TTY, Base.TTY}
 VSCodeServer.InlineDisplay
```